app-backend/database/src/main/scala/com/azavea/rf/database/Database.scala

```diff
@@ -20,6 +20,7 @@ class Database(jdbcUrl: String, dbUser: String, dbPassword: String) {
   hikariConfig.setJdbcUrl(jdbcUrl)
   hikariConfig.setUsername(dbUser)
   hikariConfig.setPassword(dbPassword)
+  hikariConfig.setLeakDetectionThreshold(3000L)
```

Comment:
I have it as a `Long` because it's listed as a `Long` in the [class definition](https://github.com/brettwooldridge/HikariCP/blob/master/src/main/java/com/zaxxer/hikari/HikariConfig.java#L63) but an integer definitely works.